webui/react/src/services/decoder.ts

```diff
@@ -269,6 +219,8 @@ export const decodeGetV1ExperimentRespToExperimentBase = (
     configRaw: config,
     endTime: exp.endTime as unknown as string,
     id: exp.id,
+    // numTrials
+    // labels
```

Comment:
**question:** loose ends?

Reply:
na here I was trying to add reminders that these are available but we're not / won't use them at this point.